features/input-submit.yml

```
@@ -1,5 +1,5 @@
 name: '<input type="submit">'
-description: The `<input type="submit">` HTML element submits its associated `<form>`.
+description: The `<input type="submit">` HTML element sends the data from its associated `<form>` to a server.
```

(non blocking comment) Intercepting a form `submit` is pretty common for various reasons, might never go to a server.

---

So maybe something like this?

```suggestion
description: The `<input type="submit">` HTML element represents a button that triggers an action on its associated `<form>`, such as sending the form data to a server.
```